cheri_compressed_cap_common.h

```diff
@@ -67,11 +67,20 @@ _CC_STATIC_ASSERT_SAME(_CC_N(FIELD_INTERNAL_EXPONENT_SIZE) + _CC_N(FIELD_EXP_NON
                            _CC_N(FIELD_EXPONENT_LOW_PART_SIZE),
                        _CC_N(FIELD_EBT_SIZE));
 
+// In order to allow vector loads and store from memory we can optionally reverse the first two fields.
 struct _cc_N(cap) {
     /* offset = cursor - base */
+#ifdef REVERSE_PESBT_CURSOR_ORDER
```

Comment:
Does this exist to match the in-memory order and optimize QEMU cap loads?
Maybe use something like _CC_MEMORY_ORDER_BIG_ENDIAN?

Reply:
It does, I was hoping to get loading caps down to to just using the 128 bit vector loads / stores that QEMU has. Are we happy that PESBT is canonically the 'big' end of a capability?

Reply:
I have made it _CC_CAP_MEMORY_ORDER_BIG_ENDIAN